metaci/build/tests/test_models.py

```diff
@@ -167,38 +166,40 @@ def test_get_flow_options_release(self):
         ] == datetime.date.today() + datetime.timedelta(days=6)
 
     def test_get_flow_options_push_sandbox_no_push_time(self):
-        build_flow = BuildFlowFactory()
-        build_flow.build.plan = PlanFactory(
-            role="push_sandbox", change_traffic_control=True
-        )
-        build_flow.build.plan.save()
-        build_flow.build.repo = RepositoryFactory(
-            default_implementation_steps=[
-                {
-                    "role": "push_sandbox",
-                    "duration": 10,
-                    "push_time": None,
-                    "start_time": 8,
-                    "start_date_offset": 0,
-                },
-            ],
-        )
-        build_flow.build.repo.save()
-        planrepo = PlanRepositoryFactory(
-            plan=build_flow.build.plan, repo=build_flow.build.repo
-        )
-        planrepo.save()
-        change_case_template = ChangeCaseTemplate()
-        change_case_template.save()
-        build_flow.build.release = Release(
-            repo=build_flow.build.repo,
-            change_case_template=change_case_template,
-        )
-        build_flow.build.release.version_number = "1.0"
-        build_flow.build.release.save()
-        options = build_flow._get_flow_options()
-        assert options["push_sandbox"]["version"] == "1.0"
-        assert options["push_sandbox"]["start_time"] is None
+        with pytest.raises(Exception):
```

Comment:
This is way too careless. There could be an exception on the very first statement and the test would pass. This should go around only the statement where the exception is expected, and you should use `with pytest.raises(Exception, match="some text")` to make sure the exception is the one we expected and not something else.